plutus-ledger-api/test-onchain-evaluation/README.md

```diff
@@ -0,0 +1,55 @@
+# Plutus On-Chain Script Evaluation Tests
+
+This test suite evaluates Plutus scripts that ran on the Cardano mainnet.
+It verifies the preservation of behavior: existing scripts that succeeded (resp. failed) on-chain on a given input should always continue to succeed (resp. fail) when evaluated on plutus master given the same input.
+
+The test suite currently runs nightly.
+
+## Generating Script Dumps
+
+The executable for generating script dumps is in the [plutus-apps](https://github.com/input-output-hk/plutus-apps) repo, since it depends on the node API.
+It is currently in the "plutus-script-dump" branch and will eventually be merged into the main branch.
```

Comment:
What's stopping it from going into `next-node`?

Reply:
It currently depends on unmerged changes of node and ledger:
- https://github.com/input-output-hk/cardano-node/pull/3984
- https://github.com/input-output-hk/cardano-node/pull/3993
- https://github.com/input-output-hk/cardano-ledger/pull/3052

Also, `next-node` currently depends on plutus release/1.0.0, but this needs a newer plutus version.

Reply:
I'll bring up https://github.com/input-output-hk/cardano-ledger/pull/3052 next week when we plan who is working on what, so that someone can spend some time thinking about a proper fix.

Reply:
@JaredCorduan thanks!